jx/jx-versioning-demo.md

```diff
@@ -29,8 +29,13 @@ jx rsh -d
 
 cd go-demo-6
 
+#For linux
```

Comment:
If we provide instructions for Mac and Linux, we need to do the same for Windows as well.